README.md

```diff
@@ -0,0 +1,50 @@
+# NevolutionX
+Original Xbox dashboard created with nxdk.
+
+![Screenshot](https://en.wikipedia.org/wiki/Hammer_and_sickle#/media/File:Hammer_and_sickle_red_on_transparent.svg "Screenshot")
+
+## Motivation
+As the XboxDev community grows, the need for an open-source, nxdk based dashboard arises. There are other dashboard alternatives i.e. XBMC but none of them are build using nxdk.
+
+## Features
+- [x] Basic Menu
+- [x] Build-in FTP server
+	- [x] IPv4 & IPv6
+	- [ ] TLS
+- [x] Application launcher
+- [x] DVD launcher
+- [ ] Themes
+
+## Build
+
+In order to build NevolutionX you'll need to install [nxdk](https://github.com/XboxDev/nxdk) first.
+To build NevolutionX you simply clone the repository and run `cmake` followed by `make`. You may need to adjust the installation location of nxdk in the `Makefile` to your needs. The default is `NXDK_DIR = $(CURDIR)/../nxdk` which expects nxdk right next to NevolutionX.
+
+```sh
+git clone https://github.com/dracc/NevolutionX.git
+cd NevolutionX/
+mkdir build
+cd build
+cmake ..
+make
+```
+
+The xbox executable(`default.xbe`) will be inside the newly created `bin/`:
+```sh
+ls bin/
+480.bmp 720.bmp default.xbe vegur.ttf
+```
+## Installation
+Simply copy and run the `default.xbe` onto your xbox. 
```

Comment:
I think something like this would look cleaner;
```diff
- Simply copy and run the `default.xbe` onto your xbox.
- Do not forget to copy the assets together with it.
+ Simply copy `default.xbe` and all the assets onto your xbox.
```

Reply:
Sounds better.